From 69479e6c67724d17953ce406c7b92f644b8c5310 Mon Sep 17 00:00:00 2001
From: zouxiaoh <xiaohong.zou@intel.com>
Date: Mon, 21 Aug 2023 17:28:39 +0800
Subject: [PATCH 1/5] media: intel: Enable LT6911UXC, LT6911UXE and Intel IPU6

Change Description:
Message for Open Source:
Enable LT6911UXC, LT6911UXE and Intel IPU6

Message for Internal:
Enable LT6911UXC, LT6911UXE and Intel IPU6

Test Platform:
ADL-N CRB

Signed-off-by: zouxiaoh <xiaohong.zou@intel.com>
---
 drivers/media/i2c/Kconfig  | 22 ++++++++++++++++++++++
 drivers/media/i2c/Makefile |  2 ++
 drivers/media/pci/Kconfig  |  2 +-
 3 files changed, 25 insertions(+), 1 deletion(-)

diff --git a/drivers/media/i2c/Kconfig b/drivers/media/i2c/Kconfig
index 226454b6a90d..fee6b96c4517 100644
--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -1623,6 +1623,28 @@ config VIDEO_THS7303
 	  To compile this driver as a module, choose M here: the
 	  module will be called ths7303.
 
+config VIDEO_LT6911UXC
+	tristate "Lontium LT6911UXC decoder"
+	depends on VIDEO_DEV && I2C
+	select VIDEO_V4L2_SUBDEV_API
+	help
+	This is a Video4Linux2 sensor-level driver for the Lontium
+	LT6911UXC HDMI to MIPI CSI-2 bridge.
+
+	To compile this driver as a module, choose M here: the
+	module will be called lt6911uxc.
+
+config VIDEO_LT6911UXE
+	tristate "Lontium LT6911UXE decoder"
+	depends on VIDEO_DEV && I2C
+	select VIDEO_V4L2_SUBDEV_API
+	help
+	This is a Video4Linux2 sensor-level driver for the Lontium
+	LT6911UXE HDMI to MIPI CSI-2 bridge.
+
+	To compile this driver as a module, choose M here: the
+	module will be called lt6911uxe.
+
 endmenu
 
 endif # VIDEO_DEV
diff --git a/drivers/media/i2c/Makefile b/drivers/media/i2c/Makefile
index c743aeb5d1ad..8fdf95b90763 100644
--- a/drivers/media/i2c/Makefile
+++ b/drivers/media/i2c/Makefile
@@ -141,3 +141,5 @@ obj-$(CONFIG_VIDEO_VP27SMPX) += vp27smpx.o
 obj-$(CONFIG_VIDEO_VPX3220) += vpx3220.o
 obj-$(CONFIG_VIDEO_WM8739) += wm8739.o
 obj-$(CONFIG_VIDEO_WM8775) += wm8775.o
+obj-$(CONFIG_VIDEO_LT6911UXC) += lt6911uxc.o
+obj-$(CONFIG_VIDEO_LT6911UXE) += lt6911uxe.o
diff --git a/drivers/media/pci/Kconfig b/drivers/media/pci/Kconfig
index 480194543d05..ee095bde0b68 100644
--- a/drivers/media/pci/Kconfig
+++ b/drivers/media/pci/Kconfig
@@ -73,7 +73,7 @@ config VIDEO_PCI_SKELETON
 	  Enable build of the skeleton PCI driver, used as a reference
 	  when developing new drivers.
 
-source "drivers/media/pci/intel/ipu3/Kconfig"
+source "drivers/media/pci/intel/Kconfig"
 
 endif #MEDIA_PCI_SUPPORT
 endif #PCI
-- 
2.34.1

